cmd/sblookup/main.go

```diff
@@ -39,8 +39,9 @@ import (
 )
 
 var (
-	apiKeyFlag   = flag.String("apikey", "", "specify your Safe Browsing API key")
-	databaseFlag = flag.String("db", "", "path to the Safe Browsing database. By default persistent storage is disabled (not recommended).")
+	apiKeyFlag    = flag.String("apikey", "", "specify your Safe Browsing API key")
+	databaseFlag  = flag.String("db", "", "path to the Safe Browsing database. By default persistent storage is disabled (not recommended).")
+	serverUrlFlag = flag.String("server", safebrowsing.DefaultServerURL, "Safebrowsing API server address")
```

Comment:
Thanks - resolved in https://github.com/google/safebrowsing/pull/46/commits/f41aa662732221094baa4600eae16e738c48ad1e